rtrlib/rtr_mgr.c

```diff
@@ -721,13 +711,13 @@ int rtr_mgr_remove_group(struct rtr_mgr_config *config,
 }
 
 // TODO: write test for this function.
-int rtr_mgr_for_each_group(struct rtr_mgr_config *conf,
+int rtr_mgr_for_each_group(struct rtr_mgr_config *config,
```

Comment:
Please sync signature with declaration in the header.

Reply:
? it matches, besides the function pointer (fp) is used here as with all other `_for_each_` functions, see below. Or what do you mean?